homeWork_01/ex2_Algorithm/VikaBakunovich/algorithms.txt

```diff
@@ -0,0 +1,42 @@
+Оплата за телефон
+1. Подойти к терминалу по оплате платежей
+2. Выбрать оператора связи
+3. Ввести номер телефона
+4. Проверить правильность введенного номера
+5. Вставить денежную купюру в купюроприёмник
+6. Дождаться сообщения о зачислении денег на счет
+7. Получить чек
+
+Действие человека при переходе улицы
+1. Подойти к переходу 
+2. Дождаться зелёного света
+3. Перейти улицу
```

Comment:
i wouldn't like to get such instruction to my child . what about the cars completing the maneuver ?